apps/explorer/app/block/[id]/page.tsx

```diff
@@ -34,21 +34,30 @@ export const revalidate = 0
 
 export default async function Page({ params }) {
   const id = params?.id as string
-  const [b, error] = await to(
-    siaCentral.block({
-      params: {
-        id,
-      },
-    })
+
+  // 1. Grab the tip at this height
```

Comment:
End the comments with a period. You may see a mix of styles in the codebase but I've been trying to move everything to sentences that end in periods, as its is consistent/works for longer comments with multiple sentences. If you come across old comments please do update them.